### PR TITLE
Add Mircoworkers

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Microworkers is a crowdsourcing platform that allows businesses or individuals to distribute small tasks to freelancers or task performers around the world. The platform provides 2FA for the account login verification process. The website is ranked 39,190 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=microworkers.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: entries/m/microworkers.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Microworkers": {
+    "domain": "microworkers.com",
+    "url": "https://www.microworkers.com",
```
Comment (hkamran80) [inline entries/m/microworkers.com.json]:
```suggestion
```

The `url` is not needed when going to `https://[domain]` works.

File: entries/m/microworkers.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "tfa": [
+      "totp",
+      "email"
+    ],
+    "documentation": "https://www.microworkers.com/blog/whats-new-in-mw-implementation-of-two-factor-authentication-2fa/",
+    "categories": [
+      "task"
```
Comment (hkamran80) [inline entries/m/microworkers.com.json]:
```suggestion
      "other"
```

The `task` category is for task management and to-do services like Airtable and Todoist. I don't think any of our existing categories fit Microworkers well.

File: img/m/microworkers.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" width="16" height="16"><image xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAIGNIUk0AAHomAACAhAAA+gAAAIDo AAB1MAAA6mAAADqYAAAXcJy6UTwAAACiUExURf////CiouyMjPS5uvCgoOlyc+ZhYfnc3Pri4u6X l+VdXv309P77++FDRN4xMt4wMeA9PuA4OeNOTvzq6uyDhN4uL+hrbORWV/bIyN0rLO6TlPfQ0fCo qaxUY9ycocbM2M4yN9UtMO7e4be2xdhBRFaQtjR2opO50Cdwn6zJ2zp/qsHX5KXE2BZnm/L3+Rlq nBhpm0OEruXu9IOuyV2Wuo+3z+sDLFgAAAABYktHRACIBR1IAAAAB3RJTUUH6AsSBCg2Lu0OpQAA AMZJREFUGNNlj912gjAQhKeKSm0kRZvW+NdahQYw2TXA+79aF4537sVcfGd3ZhZ4mpfJNJnNJ4v0 db5M3gC1ynT2rrVIrvVaQeVZvlEfxnwq9aWzzQDWwNaaBNgZnQ5gCeytnQKJMengMQMW1myBg5ET HO0J+P45/wKX81ViivJPxLkKqF0jPW6+QCDiO4L3UUD0LlS+rSk2vh6rOt90HG/ct74dQceFq2Wx LLkZQcXEvTgSURhBJCYx65m7x7/VXYIRumLI+Ad+gRTE5rM7GQAAACV0RVh0ZGF0ZTpjcmVhdGUA MjAyNC0xMS0xOFQwNDo0MDo1NCswMDowMNrCKYMAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjQtMTEt MThUMDQ6NDA6NTQrMDA6MDCrn5E/AAAAKHRFWHRkYXRlOnRpbWVzdGFtcAAyMDI0LTExLTE4VDA0 OjQwOjU0KzAwOjAw/Iqw4AAAAABJRU5ErkJggg==" width="16" height="16"/></svg>
```
Comment (hkamran80) [inline img/m/microworkers.com.svg]: Can you find an SVG that doesn't have a PNG embedded? If you can't, upload a PNG.

**Review by hkamran80 — DISMISSED** [review]

File: entries/m/microworkers.com.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/m/microworkers.com.json]: Thank you for your reply. This is my first submission, so I don’t quite understand some of the rules. I will pay more attention to it later!

File: entries/m/microworkers.com.json (hunk 2 above)
Comment (Yang-Nankai) [reply entries/m/microworkers.com.json]: I was indeed struggling with the categorization of microworkers.com, but thanks for correcting me, I will open a issue later if I am unsure!

File: img/m/microworkers.com.svg (hunk 3 above)
Comment (Yang-Nankai) [reply img/m/microworkers.com.svg]: I have done my best to find PNG images and compress them to 4KB.

Comment (hkamran80) [reply img/m/microworkers.com.svg]: @Yang-Nankai Here's an even smaller one, compressed by [TinyPNG](https://tinypng.com).

Microworkers

Comment (Yang-Nankai) [reply img/m/microworkers.com.svg]: Thanks! I have committed it again!

File: entries/m/microworkers.com.json (hunk 2 above)
Comment (Carlgo11) [reply entries/m/microworkers.com.json]: @Yang-Nankai I think you forgot to commit this change.

Comment (Yang-Nankai) [reply entries/m/microworkers.com.json]: @Carlgo11 Sorry! I'm not familiar with GitHub yet! Thanks for your reminder, I should have revised and committed it now!

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂

Your contribution will be visible in around 10 minutes. Sometimes, it takes longer to show up due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

P.S. You don't need to request  a review. Me or another maintainer will review it when we have time.

**Pull request merged**